platform/android/export/export.cpp

```diff
@@ -1786,6 +1786,13 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			}
 		}
 
+		String rk = p_preset->get("keystore/release");
+
+		if (!rk.empty() && !FileAccess::exists(rk)) {
+			valid = false;
```

Comment:
The thing is that the absence of release keystore shouldn't prevent users from exporting debug builds.
That's why one can configure an editor-wide debug keystore, which lets you export any project in debug mode without having to care about the signing.

Reply:
Or nevermind, the check for `!rk.empty()` already ensures that this will only run if a release keystore has been defined, but is invalid. It's good then.

Reply:
Yes exactly, thanks!